yaml-test-suite/test_suite_test.go

```diff
@@ -0,0 +1,230 @@
+package yaml_test
```

Comment:
The package name collides with the one in the main package, I suggest naming it `yts`. If the intention is to move all the tests from main to a dedicated dir, then let's rename `yaml-test-suite` to just `test` and `test_suite_test.go` to `yaml_suite_test.go`.

Reply:
I applied your name change. If we want to move the files we can move in another PR.